lms/djangoapps/dashboard/tests/test_sysadmin.py

```diff
@@ -582,6 +583,38 @@ def test_gitlog_no_logs(self):
 
         self._rm_edx4edx()
 
+    def test_gitlog_pagination_out_of_range_invalid(self):
+        """
+        Make sure the pagination behaves properly when the requested page is out
+        of range.
+        """
+
+        self._setstaff_login()
+
+        mongoengine.connect(TEST_MONGODB_LOG['db'])
+
+        for x in xrange(15):
+            CourseImportLog(
+                course_id=SlashSeparatedCourseKey("test", "test", "test"),
+                location="location",
+                import_log="import_log",
+                git_log="git_log",
+                repo_dir="repo_dir",
+                created=datetime.now()
+            ).save()
```

Comment:
It appears a lack of a delete/teardown here is causing a unit test to fail elsewhere.  I'm not sure where you want to fix the issue, but probably just a teardown that nukes all `CourseImportLog` entries in this test class makes sense to me